tests/test_autonomy/test_cli/test_mint/test_mint_components.py

```diff
@@ -46,24 +47,43 @@ def get_create_transaction(self, *args: Any, **kwargs: Any) -> None:
 
 @skip_docker_tests
 @pytest.mark.usefixtures("registries_scope_class")
-class TestMintProtocol(BaseCliTest):
-    """Test `autonomy develop mint protocol` command."""
+class TestMintComponents(BaseCliTest):
+    """Test `autonomy develop mint` command."""
 
-    cli_options = ("mint", "protocol")
+    cli_options = ("mint",)
 
     def setup(self) -> None:
         """Setup test."""
         super().setup()
         self.cli_runner.mix_stderr = False
 
-    def test_mint(
-        self,
-    ) -> None:
+    @pytest.mark.parametrize(
+        argnames=("package_type", "package_path"),
+        argvalues=(
+            (PackageType.PROTOCOL, "packages/valory/protocols/acn"),
+            (PackageType.CONTRACT, "packages/valory/contracts/service_registry"),
+            (PackageType.CONNECTION, "packages/valory/connections/abci"),
+            (PackageType.SKILL, "packages/valory/skills/abstract_round_abci"),
+            (PackageType.AGENT, "packages/valory/agents/hello_world"),
+        ),
+    )
+    def test_mint(self, package_type: PackageType, package_path: str) -> None:
         """Test mint protocol."""
 
-        result = self.run_cli(
-            commands=("packages/valory/protocols/acn", str(ETHEREUM_KEY_DEPLOYER)),
-        )
+        if package_type == PackageType.AGENT:
+            result = self.run_cli(
+                commands=(
+                    package_type.value,
+                    package_path,
+                    str(ETHEREUM_KEY_DEPLOYER),
+                    "-d",
+                    "1",
```

Comment:
no magic values

Reply:
This is a roughly put together test, I'll update the test properly once we're done with the command implementation